org.megamek.MegaMek.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: org.megamek.MegaMek
+runtime: org.freedesktop.Platform
+runtime-version: 22.08
+sdk: org.freedesktop.Sdk
+command: megamek
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+finish-args:
+  - --socket=wayland

```

Comment:
does it really work with wayland?

My experience with Java is that usually it doesn't.

Reply:
I haven't tried. It uses the Swing UI toolkit.